maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -59,6 +62,7 @@
 import org.apache.maven.project.MavenProject;
 import org.apache.maven.shared.artifact.filter.PatternIncludesArtifactFilter;
 import org.apache.maven.shared.transfer.dependencies.resolve.DependencyResolver;
+import org.apache.maven.surefire.shared.utils.StringUtils;
```

Comment:
Pls remove this line, because there already was this import line:
`import static org.apache.maven.surefire.shared.utils.StringUtils.isNotBlank;`